### PR TITLE
motd: link to FCOS Discourse forum

### PR DESCRIPTION
**Pull request by jlebon**
Add a link to the Discourse forum directly in the MOTD to make it easier
for new users to figure out where to discuss and ask for help.

### CODE REVIEW

**Comment by jlebon** [comment]
Related: https://github.com/coreos/fedora-coreos-docs/pull/41

File: overlay.d/15fcos/usr/lib/motd.d/tracker.motd [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Tracker: https://github.com/coreos/fedora-coreos-tracker
+Discuss: https://discussion.fedoraproject.org/c/server/coreos/
```
Comment (dustymabe) [inline overlay.d/15fcos/usr/lib/motd.d/tracker.motd:2]: Do the words `Tracker` and `Discuss` give the user enough info to make a decision? before we only had one option so `Tracker` didn't really matter. but now that we have two should we clarify more? I'm torn

**Review by dustymabe — APPROVED** [review]
LGTM - one comment

File: overlay.d/15fcos/usr/lib/motd.d/tracker.motd (hunk 1 above)
Comment (cgwalters) [reply overlay.d/15fcos/usr/lib/motd.d/tracker.motd:2]: Let's `Discuss`!

**Review by cgwalters — APPROVED** [review]

File: overlay.d/15fcos/usr/lib/motd.d/tracker.motd (hunk 1 above)
Comment (jlebon) [reply overlay.d/15fcos/usr/lib/motd.d/tracker.motd:2]: I guess one other option is `Questions`? But then the colons won't line up! :( And it doesn't convey the fact you could just chat there about how awesome FCOS is without actually having a question. :)

Not sure, inclined to just ship as is.

Comment (dustymabe) [reply overlay.d/15fcos/usr/lib/motd.d/tracker.motd:2]: 👍

**Pull request merged**